www/caddy/src/opnsense/scripts/OPNsense/Caddy/caddy_certs.php

```diff
@@ -27,61 +27,83 @@
  *    POSSIBILITY OF SUCH DAMAGE.
  */
 
+namespace OPNsense\Caddy;
```

Comment:
you should be able to ditch the namespace here and import Caddy for the required modules, as it's not a module being imported elsewhere.